src/sql/parts/connection/common/connectionContextKey.ts

```diff
@@ -33,12 +36,16 @@ export class ConnectionContextKey implements IContextKey<IConnectionProfile> {
 		this._connectionKey.set(value);
 		this._providerKey.set(value && value.providerName);
 		this._serverKey.set(value && value.serverName);
+		let majorVersion = value && value.options && value.options['serverMajorVersion'];
+		this._serverMajorVersion.set(majorVersion && `${majorVersion}`);
```

Comment:
Does this convert from number to string?

Reply:
Yes. `'${majorVersion}'` returns string.